packages/pds/src/account-manager/helpers/used-refresh-token.ts

```diff
@@ -0,0 +1,24 @@
+import { RefreshToken } from '@atproto/oauth-provider'
+import { AccountDb } from '../db'
+
+export const insert = async (
+  db: AccountDb,
+  id: number,
+  usedRefreshToken: RefreshToken,
+) => {
+  await db.executeWithRetry(
+    db.db
+      .insertInto('used_refresh_token')
```

Comment:
Is the idea that `used_refresh_token` gets cleaned-up by virtue of cleanup in `token` causing cascading deletes?   If that is the case, it may be worth documenting in a comment somewhere.